lib/resources/r4/document_reference.yaml

```diff
@@ -29,3 +116,143 @@ fields:
     - display: US Core DiagnosticReport Category
       system: http://loinc.org
       info_link: http://hl7.org/fhir/us/core/ValueSet/us-core-diagnosticreport-category
+
+- name: subject
+  required: 'Yes'
+  type: Reference
+  action:
+    - create
+  description: Who the document is about.
+  note: Only patient references are supported.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.subject
+  example: |
+    {
+      "subject": {
+        "reference": "Patient/1415936"
+      }
+    }
+
+- name: author
+  required: 'Yes'
+  type: Reference
+  action:
+    - create
+  description: Identifies who is responsible for adding the information to the document.
+  note: Must contain only Practitioner references with valid millennium ids.  Optional for system access.  For provider access, only a single author can be provided, and it must be the authorized provider.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author
+  example: |
+    {
+      "author": [
+        {
+          "reference": "Practitioner/123456"
+        }
+      ]
+    }
+
+- name: authenticator
+  required: 'No'
+  type: Reference
+  action:
+    - create
+  description: Which person or organization authenticates that this document is valid.
```

Comment:
```suggestion
  description: Which person authenticates that this document is valid.
```

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/1560e66bcece049dcfee1e781105538fe4d7898f

Reply:
I don't think change was made either.